Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_DarkBeast.xml

```diff
@@ -14,7 +14,21 @@
 						</li>
 					</value>
 				</li>
-				
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_Darkbeast"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>4</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_Darkbeast"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>2</ArmorRating_Sharp>
```

Comment:
The vanilla version of this creature has extremely little armor, and it's CE variants have none. At most, I'd give this 0.2 sharp and 0.4 blunt armor, if I were giving it armor at all.